packages/data-context/src/sources/RelevantRunSpecsDataSource.ts

```diff
@@ -153,22 +139,38 @@ export class RelevantRunSpecsDataSource {
         statuses: {},
       }
 
-      if (cloudProject.current && cloudProject.current.runNumber && cloudProject.current.status) {
+      const { current, next } = cloudProject
+
+      if (
+        current
+         && current.runNumber
+         && current.status
+         && typeof current.totalInstanceCount === 'number'
```

Comment:
You could use `Number.isFinite(...)` as well to save a few chars and exclude edge cases like `*Ininity` and `NaN`

Reply:
Cool, sure 👍